### ISSUE TITLE
A quarter of offest?

### ISSUE THREAD

**Issue by hellojialee** [anewell/pose-hg-demo#16]
In the original paper, there goes: "the prediction is offset by a quarter of a pixel in the direction of its next highest neighbor before transforming back to the original coordinate space of the image"

Is it implemented in this project? Looking forward for replying. Thank you!

**Comment by anewell** [comment]
I don't think it is included here, but you can find the code for it in the training repo here: https://github.com/umich-vl/pose-hg-train/blob/master/src/util/pose.lua#L85

**Comment by hellojialee** [comment]
@anewell Thank you for your kind replying!  I'm not familiar with lua. Do you simply find the max response (regarded as a pixel value, let's denote it **P**) on the heat map and then shift towards to the second max response which is within its 4-neighbors or just another max response of a gussian circle?
![image](https://user-images.githubusercontent.com/21237799/35020852-0499a686-fb69-11e7-9b5e-f71a0818f633.png)
